tests/Integration/Tagging/Fixtures/get_asset_route.php

```diff
@@ -0,0 +1,9 @@
+<?php declare(strict_types=1);
+
+use App\Controller\GetAssetController;
+use Symfony\Component\Routing\Loader\Configurator\RoutingConfigurator;
+
+return function (RoutingConfigurator $routes) {
+    $routes->add('get_asset', '/get-asset')
+        ->controller([GetAssetController::class, '__invoke']);
```

Comment:
This should be enough, I support:

```suggestion
        ->controller(GetAssetController::class);
```

Same for the other routes.